modules/planning/common/frame.h

```diff
@@ -169,6 +170,13 @@ class Frame {
 
   perception::TrafficLight GetSignal(const std::string &traffic_light_id) const;
 
+  std::tuple<bool, double, double, double> pull_over_info() {
+    return pull_over_info_;
+  }
+
+  void set_pull_over_info(const std::tuple<bool, double, double, double>&
```

Comment:
Where is this "set_pull_over_info" called? Also is the tuple defined as "is_feasible_widnow, pull_over_x, pull_over_y, pull_over_phi"? If so, could you add comments here so to avoid confusion?

Reply:
Thanks. This problem is solved. Please see the most recent commit.